src/standardValueSetTranslations/Attribute__cSubject_Area__c-ja.standardValueSetTranslation

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is a duplicate of the values for the Subject_Area__c field on the Attribute__c Object Translations file [here](https://github.com/SalesforceFoundation/EDA/blob/d2f811c912f766203c2a32e90c2d32cdd2d7c5a6/src/objectTranslations/Attribute__c-ja.objectTranslation#L120). Please confirm this is the desired state and whether we want to have duplicated translations.